### DISCUSSION TITLE
Encoding output?

### DISCUSSION THREAD

**Discussion by chrstffr** [gorilla-devs/ferium#153] [Q&A] [answered]
I'm a complete noob when it comes to contemporary Windows, been using Linux and macOS for the last 20+ years. Not sure what this problem is about, but I get strange output from ferium.exe both in cmd.exe and Windows PowerShell. This is what it looks like:

```
←[1mLitematica                                   ←[0m ←[31mCurseForge←[0m ←[2m308892←[0m
←[1mMaLiLib                                      ←[0m ←[31mCurseForge←[0m ←[2m303119←[0m
←[1mMiniHUD                                      ←[0m ←[31mCurseForge←[0m ←[2m244260←[0m
←[1mMod Menu                                     ←[0m ←[31mCurseForge←[0m ←[2m308702←[0m
←[1mQuilt Standard Libraries                     ←[0m ←[32mModrinth  ←[0m ←[2mqvIfYCYJ←[0m
```

I've googled a bit about locale/encoding in the shell and that led me to try the global utf-8 support option, but that didn't help. Just changed some of the characters to the missing utf8 char icon.

Is there a way to fix this?

**Comment by theRookieCoder** [comment, marked as answer]
#149

Basically the terminal colour library I use doesn't support conhost, i.e. the cmd and powershell  apps. I highly recommend you just use windows terminal

**Reply by chrstffr** [reply]
Thank you so much, I didn't even know that was a thing. 🙏
